cpp/community_detection_module/grappolo/DefineStructure/coloring.h

```diff
@@ -14,7 +14,7 @@ int algoColoringMultiHashMaxMin(graph *G, int *vtxColor, int nThreads, double *t
 // In vBase.cpp
 int vBaseRedistribution(graph* G, int* vtxColor, int ncolors, int type);
 
-// In equtiableColoringDistanceOne.cpp
+// In equitableColoringDistanceOne.cpp
```

Comment:
The filename is correctly spelled in version control, only this reference was wrong...